src/EPR.PRN.Backend.API.UnitTests/Repositories/RepositoryTestsInMemory.cs

```diff
@@ -465,4 +465,84 @@ private static Eprn CreateEprn(Guid orgId, string prnNumber, string issuedByOrg,
             CreatedOn = DateTime.Now
         };
     }
+
+    private static Eprn CreateEprnEntityFromDto(SavePrnDetailsRequest prn)
+    {
+        if (prn == null) return null;
+
+        Eprn prnEntity = new Eprn()
+        {
+            AccreditationNumber = prn.AccreditationNo!,
+            AccreditationYear = prn.AccreditationYear.ToString()!,
+            // CancelledDate = prn.CancelledDate, // This property /column does not exist on Eprn entity or DB table PRN
```

Comment:
should we remove commented code?

Reply:
like I said below - this property has no defined mapping in the mapping document so, I left it there with meaningful comment because it could get missed out otherwise